CHANGELOG.md

```diff
@@ -3,6 +3,81 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.18.0
+
+_Sep 19, 2024_
+
+We'd like to offer a big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
```

Comment:
Any suggestions for highlights?

Reply:
```
- [charts] Add a `PolarProvider` to manage polar axes (#14642) @alexfauquette
```

Maybe, if @alexfauquette thinks it's worth highlighting